apps/src/templates/teacherDashboard/SetUpMessage.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import i18n from "@cdo/locale";
+import color from "../../util/color";
+import ProgressButton from "../progress/ProgressButton";
+
+const styles = {
+  section: {
+    width: 1075,
+    border: '3px dashed gray',
+    backgroundColor: color.white,
+    marginLeft: 35
+  },
+  heading: {
+    paddingLeft: 25,
+    paddingTop: 25,
+    fontSize: 30,
+    fontFamily: '"Gotham 5r", sans-serif',
+    color: color.teal,
+  },
+  description: {
+    paddingLeft: 25,
+    paddingTop: 25,
+    paddingBottom: 25,
+    fontSize: 18,
+    color: color.charcoal
+  },
+  button: {
+    marginLeft: 25,
+    marginBottom: 25
+
+  }
+};
+
+const SetUpMessage = React.createClass({
+  propTypes: {
+    type: React.PropTypes.oneOf(['courses', 'sections']).isRequired
+  },
+
+  render() {
+    const { type } = this.props;
+
+    if (type === 'courses') {
```

Comment:
You could do something like
```jsx
const isCourses = (type === 'courses');
return (
  <div style={styles.section}>
    <div style={styles.heading}>
      {isCourses ? i18n.startLearning() : i18n.setupClassroom)}
  </div>
//etc
```
In this case I think I'm kind of okay with either approach.